src/redux/greetings/greetings.js

```diff
@@ -0,0 +1,31 @@
+const URL = 'http://localhost:4200/greetings';
+
+// Action types
+const LOAD_GREETING = 'LOAD_GREETING';
+
+const initialState = {
+  message: '',
+};
+
+// Reducers
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case LOAD_GREETING:
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+// Action Creators
+export const loadGreeting = () => async (dispatch) => {
+  const res = await fetch(URL, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  });
+  const payload = await res.json();
+  dispatch({ type: LOAD_GREETING, payload });
+};
```

Comment:
**Changes Required**
- [ ] Good effort with redux. However, actions should be a file of it's own. This will allow us to follow the `SOLID` principle with the `S` being Separation of Concerns or Single Responsibility. Something that you already learned in your previous lessons in Ruby. Let us please refactor this to keep our code base clean.